server/public/model/metrics.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/blang/semver/v4"
+)
+
+type MetricType string
+
+const (
+	ClientTimeToFirstByte        MetricType = "time_to_first_byte"
+	ClientFirstContentfulPaint   MetricType = "first_contentful_paint"
+	ClientLargestContentfulPaint MetricType = "largest_contentful_paint"
+	ClientInteractionToNextPaint MetricType = "interaction_to_next_paint"
+	ClientCumulativeLayoutShift  MetricType = "cumulative_layout_shift"
```

Comment:
Sinve these names are only used when getting values from the frontend and aren't even passed to the metrics code, I'd rather have these use the names directly from the Web Vitals library to avoid having to map them from the Web Vitals name to our name and then from our name to the function handler
```suggestion
	ClientTimeToFirstByte        MetricType = "TTFB"
	ClientFirstContentfulPaint   MetricType = "FCP"
	ClientLargestContentfulPaint MetricType = "LCP"
	ClientInteractionToNextPaint MetricType = "INP"
	ClientCumulativeLayoutShift  MetricType = "CLS"
```

The rest can stay as-is since we dictate those ones